test/connect.spec.js

```diff
@@ -495,6 +508,44 @@ describe('connect', () => {
     })
     assert(wrapped.$el.textContent === 'foo value,bar value')
   })
+
+  it('handles namespaced state in stateToProps', () => {
+    const Container = connect({
+      stateToProps: (mapState) => ({
+        ...mapState('inner_foo', {
+          namespaced_foo: 'foo',
+        }),
+        ...mapState({
+          foo: 'foo'
+        })
+      }),
+      mixins: {}
```

Comment:
What is this and [this](https://github.com/ktsn/vuex-connect/pull/58/files#diff-35bd9385241b62924b957d3ef010c2faR541) line for?

Reply:
Definitely a brainfart, unnecessary. Removed